frontend/taipy-gui/src/components/Taipy/Pane.tsx

```diff
@@ -133,6 +160,12 @@ const Pane = (props: PaneProps) => {
                 </>
             </Tooltip>
         </Drawer>
+    ) : showButton ? (
+        <Drawer variant="permanent" sx={buttonDrawerSx} anchor={anchor} open={true} className={getSuffixedClassNames(className, "-button")}>
```

Comment:
Can you think of a way to find, in the code, all (or most of) those cases where computed classes were added to sub-components?